tests/migrations/test_autodetector.py

```diff
@@ -2181,6 +2181,155 @@ def test_rename_model_reverse_relation_dependencies(self):
             changes, "testapp", 0, 0, old_name="EntityA", new_name="RenamedEntityA"
         )
 
+    def test_move_model(self):
+        before = [
+            ModelState(
+                "cars",
+                "Car",
+                [
+                    ("id", models.AutoField(primary_key=True)),
+                    ("price", models.IntegerField(default=0)),
+                ],
+            ),
+            ModelState(
+                "cars",
+                "TyreBrand",
+                [
+                    ("id", models.AutoField(primary_key=True)),
+                    ("name", models.CharField(max_length=255)),
+                ],
+            ),
+            ModelState(
+                "tires",
+                "Tyre",
+                [
+                    ("id", models.AutoField(primary_key=True)),
+                    ("size", models.CharField(default=16)),
+                    ("brand", models.ForeignKey("cars.TyreBrand", models.CASCADE)),
+                ],
+            ),
```

Comment:
Can we reuse existing test states? For example:
```suggestion
            self.publisher,
            self.author_with_publisher,
```
and change them in `after`.